src/js/components/Tabs/Tabs.js

```diff
@@ -9,109 +9,98 @@ import { Box } from '../Box';
-      theme,
-      ...rest
-    } = this.props;
-    delete rest.activeIndex;
-    delete rest.onActive;
-    const { activeIndex } = this.state;
-
-    let activeContent;
-    let activeTitle;
-    const tabs = Children.map(
-      children,
-      (tab, index) => {
-        if (!tab) return undefined;
-
-        const tabProps = tab.props || {};
-
-        const isTabActive = index === activeIndex;
-
-        if (isTabActive) {
-          activeContent = tabProps.children;
-          if (typeof tabProps.title === 'string') {
-            activeTitle = tabProps.title;
-          } else {
-            activeTitle = index + 1;
-          }
-        }
+  /* eslint-disable no-param-reassign */
+  delete rest.activeIndex;
+  delete rest.onActive;
+  /* eslint-enable no-param-reassign */
 
-        return cloneElement(tab, {
-          active: isTabActive,
-          onActivate: () => this.activateTab(index),
-        });
-      },
-      this,
-    );
+  let activeContent;
+  let activeTitle;
+  const tabs = Children.map(
+    children,
+    (tab, index) => {
+      if (!tab) return undefined;
 
-    const tabContentTitle = `${activeTitle || ''} ${tabContents}`;
+      const tabProps = tab.props || {};
 
-    return (
-      <StyledTabs
+      const isTabActive = index === activeIndex;
+
+      if (isTabActive) {
+        activeContent = tabProps.children;
+        if (typeof tabProps.title === 'string') {
+          activeTitle = tabProps.title;
+        } else {
+          activeTitle = index + 1;
+        }
+      }
+
+      return cloneElement(tab, {
+        active: isTabActive,
+        onActivate: () => activateTab(index),
+      });
+    },
+    this,
+  );
+
+  const tabContentTitle = `${activeTitle || ''} ${tabContents}`;
+
+  return (
+    <StyledTabs
+      as={Box}
+      role="tablist"
+      flex={flex}
+      {...rest}
+      background={theme.tabs.background}
+    >
+      <StyledTabsHeader
         as={Box}
-        role="tablist"
-        flex={flex}
-        {...rest}
-        background={theme.tabs.background}
+        direction="row"
+        justify={justify}
+        flex={false}
+        wrap
+        background={theme.tabs.header.background}
+        gap={theme.tabs.gap}
       >
-        <StyledTabsHeader
-          as={Box}
-          direction="row"
-          justify={justify}
-          flex={false}
-          wrap
-          background={theme.tabs.header.background}
-          gap={theme.tabs.gap}
-        >
-          {tabs}
-        </StyledTabsHeader>
-        <StyledTabPanel
-          flex={flex}
-          aria-label={tabContentTitle}
-          role="tabpanel"
-        >
-          {activeContent}
-        </StyledTabPanel>
-      </StyledTabs>
-    );
-  }
-}
+        {tabs}
+      </StyledTabsHeader>
+      <StyledTabPanel flex={flex} aria-label={tabContentTitle} role="tabpanel">
+        {activeContent}
+      </StyledTabPanel>
+    </StyledTabs>
+  );
+};
+
+Tabs.defaultProps = {
+  justify: 'center',
```

Comment:
Can we please move this default value to line #15?

Reply:
Done.